src/components/ArticleDetails.jsx

```diff
@@ -0,0 +1,56 @@
+import { useParams } from "react-router-dom";
+import { useEffect, useState } from "react";
+import { getArticleById, getComments } from "../../api";
+import Comments from "./Comments";
+export const ArticleDetails = () => {
+  const { article_id } = useParams();
+  const [commentToggle, setCommentToggle] = useState(false);
+  const [article, setArticle] = useState([]);
+  const [comments, setComments] = useState([]);
+  useEffect(() => {
+    getArticleById(article_id)
+      .then((response) => {
+        setArticle(response);
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+  }, []);
+
+  useEffect(() => {
+    getComments(article_id)
+      .then((response) => {
+        setComments(response);
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+  }, []);
```

Comment:
Does this need to happen in this component? Or can it be moved down a level to where it is actually being used?